components/common/posts/PostsFeed.tsx

```diff
@@ -331,6 +333,67 @@ export function RecordEmbedView({
         </div>
       </div>
     )
+  } else if (AppBskyGraphDefs.isListView(embed.record)) {
+    const { did, rkey } = parseAtUri(embed.record.uri) || {}
+    const peekLink = buildBlueSkyAppUrl({
+      did: `${did}`,
+      rkey: `${rkey}`,
```

Comment:
Is this a way of casting `did` and `rkey` into a string?  If so, could it end-up casting them to something we don't intend like `'null'` or `'undefined'`?

Reply:
record.uri is always present and parsing the uri should always return did and rkey for isViewBlocked AFAICT so this is just appeasing the ts linter.